packages/iTwinUI-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -1,38 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 import cx from 'classnames';
 import React from 'react';
 import { Menu, MenuProps } from '../Menu';
-import { useSafeContext, useMergedRefs } from '../utils';
+import { Surface } from '../Surface';
+import {
+  useSafeContext,
+  useMergedRefs,
+  useVirtualization,
+  VirtualScrollProps,
+  mergeRefs,
+  getWindow,
+} from '../utils';
 import { ComboBoxStateContext, ComboBoxRefsContext } from './helpers';
 
 type ComboBoxMenuProps = Omit<MenuProps, 'onClick'> &
   React.ComponentPropsWithoutRef<'ul'>;
 
+const VirtualComboboxMenu = ({ children, ...props }: VirtualScrollProps) => {
```

Comment:
nit: can you move this to the bottom of the file? I think the exported component should come first

My preferred order is this:
```
imports
constants
types
named exports
helpers // <-- VirtualComboBoxMenu goes here
default export
```

But it's a nit, so upto you if you have a different preference.

Reply:
also why is it called `VirtualComboboxMenu` and not `VirtualizedComboBoxMenu`?

Reply:
I left it at the top as I usually put helpers before the usage. We have multiple files with this order, but it is really preference, not a set rule.
I have renamed the component.